contracts/contracts/UmbraHookable.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.6.12;
+
+/// @dev Interface that post-withdraw hooks must implement to interop with Umbra
+interface UmbraHookable {
```

Comment:
I'm very open to naming suggestions here as `UmbraHookable` was my the least-worst option I could come up with. That said, I intentionally avoided `recipient` because it felt overloaded and collided with other concepts in the protocol, i.e. the receiver of the stealth payment.

As for `callHook`, now that I think about it, using the past test description of the action that occurred is probably better. So in this case that would be something like `tokensWithdrawn` I think.

Reply:
True, good point on the overloaded use of "recipient". I do like the `tokensWithdrawn` name, but am stuck coming up with other names for `UmbraHookable`. @wildmolasses help!

Reply:
Let's also add an `I` prefix to the interface name

Reply:
Renamed to `IUmbraHookReceiver`